HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+=======
```

Comment:
I see you have a similar file here to the CHANGELOG.md one, might be worth keeping only the changelog to align with other projects?